apps/wallet/src/ui/app/redux/slices/transaction-requests/index.ts

```diff
@@ -79,10 +79,12 @@ export const deserializeTxn = createAsyncThunk<
         const signer = api.getSignerInstance(keypairVault.getKeyPair());
         const localSerializer = new LocalTxnDataSerializer(signer.provider);
         const txnBytes = new Base64DataBuffer(serializedTxn);
+        const version = await api.instance.fullNode.getRpcApiVersion();
 
         //TODO: Error handling - either show the error or use the serialized txn
         const deserializeTx =
             (await localSerializer.deserializeTransactionBytesToSignableTransaction(
+                version?.major === 0 && version?.minor > 17,
```

Comment:
ideally we should deprecate the flag when 0.18 is released. in case this flag is not removed, this should really be >= 0. need to fix in couple places (following up a PR)

Reply:
@joyqvq , 0.18.0 has been released today, we should update this number to `> 18`

Reply:
We actually should use [femver](https://www.npmjs.com/package/@suchipi/femver) to express this as `>0.17.0`